amp/TEMPLATE/ampTemplate/site-config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <!DOCTYPE page [
-<!ENTITY ampVersion "${project.version}">
+<!ENTITY ampVersion "2.13.5-SNAPSHOT">
```

Comment:
this doesn't need to be changed. in 2.13.5 this is replaced by the build process

Reply:
This needs to be resolved before merging or approving the ticket